.htaccess

```diff
@@ -29,8 +29,8 @@ ErrorDocument 500 /assets/error-500.html
 <IfModule mod_rewrite.c>
 	RewriteEngine On
 
-	RewriteCond %{HTTP_HOST} ^docs?.silverstripe.com$ [NC]
-    RewriteRule ^(.*)$ https://docs.silverstripe.org/$1 [L,R=301]
+	RewriteCond %{HTTP_HOST} ^docs?.silverstripe.org$ [NC]
```

Comment:
Ahh...I see what you are are wanting .....your right about the infinite redirects....what I had was ok when the "s?" was removed....but now it loops...oopsie.

So, you want the .com and the "s?" back in ?